src/index.js

```diff
@@ -261,7 +268,7 @@ class SwipeableViews extends React.Component {
       }
 
       return (
-        <div ref={ref} style={styles.slide}>
+        <div ref={ref} style={Object.assign(styles.slide, slideStyle)}>
```

Comment:
Can we move the `Object.assign` outside of the loop, like just before the `childrenToRender`?
Also use `Object.assign({}, styles.slide, slideStyle)` instead.